source/common/runtime/runtime_features.cc

```diff
@@ -67,6 +67,7 @@ RUNTIME_GUARD(envoy_reloadable_features_lua_flow_control_while_http_call);
 RUNTIME_GUARD(envoy_reloadable_features_mmdb_files_reload_enabled);
 RUNTIME_GUARD(envoy_reloadable_features_no_extension_lookup_by_name);
 RUNTIME_GUARD(envoy_reloadable_features_no_timer_based_rate_limit_token_bucket);
+RUNTIME_GUARD(envoy_reloadable_features_oauth2_enable_state_nonce);
```

Comment:
`envoy_reloadable_features_oauth2_enable_json_format_state`

Reply:
The runtime guard has been removed because we don't want to retain the previous behavior (url-encoded url + nonce) which caused issue [AWS cognito doesn't support url encoded state value](https://docs.aws.amazon.com/cognito/latest/developerguide/authorization-endpoint.html#get-authorize-request-parameters)